### DISCUSSION TITLE
React query view source link broken

### DISCUSSION THREAD

**Discussion by evergreenx** [TanStack/query#4857] [General] [answered]
Most of the links examples on react query docs are broken. 
[https://react-query-v3.tanstack.com/examples/custom-hooks](Docs URL)
[https://github.com/TanStack/query/tree/main/examples/custom-hooks](broken link)

![404](https://user-images.githubusercontent.com/39932962/214045295-30d536dc-449e-45dc-918d-90c8a6c0de14.PNG)

**Comment by louis-young** [comment, marked as answer]
Hi 👋 

Thanks for letting us know. This has already been raised and fixed under https://github.com/TanStack/query/issues/4108, it's just waiting to be deployed.
